docs/Using Randomness/v1/get-a-random-number.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: nodes.liquid
+section: smartContract
+date: Last Modified
+title: "Get a Random Number"
+permalink: "docs/get-a-random-number/v1/"
+whatsnext: {"API Reference":"/docs/chainlink-vrf-api-reference/v1/", "Contract Addresses":"/docs/vrf-contracts/v1/"}
+metadata: 
+  description: "How to generate a random number inside a smart contract using Chainlink VRF."
+  image: 
+    0: "/files/OpenGraph_V3.png"
+---
+This page explains how to get a random number inside a smart contract using Chainlink VRF.
+
+
+
+# Random Number Consumer
+
+Chainlink VRF follows the [Request & Receive Data](../../request-and-receive-data/) cycle. To consume randomness, your contract should inherit from <a href="https://github.com/smartcontractkit/chainlink/blob/master/evm-contracts/src/v0.6/VRFConsumerBase.sol" target="_blank">`VRFConsumerBase`</a> and define two required functions
+
+1. `requestRandomness`, which makes the initial request for randomness.
+2. `fulfillRandomness`, which is the function that receives and does something with verified randomness.
```

Comment:
```suggestion
- `requestRandomness`, which makes the initial request for randomness.
- `fulfillRandomness`, which is the function that receives and does something with verified randomness.
```

This can be unordered.